pkg/network/protocols/http/stats.go

```diff
@@ -10,9 +10,13 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/network/types"
 	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/intern"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// Interner is used to intern strings to save memory allocations.
+var Interner = intern.NewStringInterner()
```

Comment:
Does this need to be exported?

Reply:
yes, unfortunately it is used by some tests that are not part of the `http` pkg
([example](https://github.com/DataDog/datadog-agent/blob/valeri.pliskin/statkeeper-string-interning/pkg/network/encoding/encoding_test.go#L803))

Reply:
Can you explain why you need to use the same interner object? Maybe you can use a different (local) one for tests?